wrappingGsdk/wrapper/Program.cs

```diff
@@ -38,8 +44,42 @@ static void Main(string[] args)
         // starts main game process and wait for it to complete
         public static void InitiateAndWaitForGameProcess(string gameserverExe, IEnumerable<string> args)
         {
-             // here we're starting the script that initiates the game process
-            gameProcess = StartProcess(gameserverExe, args);
+            activeConfig = GameserverSDK.getConfigSettings();
+
+            // here we're starting the script that initiates the game process
+            if (activeConfig.TryGetValue(portName, out string listeningPortString))
+            {
+                GameserverSDK.LogMessage($"{portName}:{listeningPortString} was found in GSDK Confg Settings.");
+                _listeningPort = listeningPortString;
+            }
+            else
+            {
+                LogMessage($"Cannot find {portName} in GSDK Config Settings. Please make sure the MockAgent is running " +
+                             $"and that the MultiplayerSettings.json file includes correct {portName} as a GamePort Name.");
+                return;
+            }
+
+            Process processPortInUse = GetProcessPortInUse();
+
+            // port in use, something is wrong - terminating wrapper.
+            if (processPortInUse != null)
```

Comment:
can we make this if {...} a different method? Like CheckIfPortIsUsed and if returns true, it exits the app? It is not necessary but I feel it will make the sample more clear